### PR TITLE
Fix build without pulseaudio.

### PR DESCRIPTION
**Pull request by orbea**
Fixes the build when pulseaudio is not installed. Sound when playing games still works fine.

### CODE REVIEW

File: pcsx2/CMakeLists.txt [hunk 1]
```diff
@@ -953,6 +958,10 @@ if(PORTAUDIO_FOUND)
     set(pcsx2FinalLibs ${pcsx2FinalLibs} ${PORTAUDIO_LIBRARIES})
 endif()
 
+if(PULSEAUDIO_FOUND)
+    set(pcsx2FinalLis ${pcsx2FinalLibs} ${PULSEAUDIO_LIBRARIES})
```
Comment (GovanifY) [inline pcsx2/CMakeLists.txt:962]: Probably wanted to use pcsx2FinalLibs there?

**Review by GovanifY — CHANGES_REQUESTED** [review]
See comment above

File: pcsx2/CMakeLists.txt (hunk 1 above)
Comment (orbea) [reply pcsx2/CMakeLists.txt]: Thanks, good catch.

File: cmake/SearchForStuff.cmake [hunk 2]
```diff
@@ -203,7 +203,6 @@ if(PULSEAUDIO_FOUND)
     include_directories(${PULSEAUDIO_INCLUDE_DIR})
 endif()
 
-
```
Comment (GovanifY) [inline cmake/SearchForStuff.cmake:206]: Why remove this line?

**Review by GovanifY — COMMENTED** [review]
Sorry, didn't catch that before, otherwise LGTM

File: cmake/SearchForStuff.cmake (hunk 2 above)
Comment (orbea) [reply cmake/SearchForStuff.cmake:206]: Extra blank lines are a pet peeve of mine and I removed it by habit when looking at the above lines. I can revert this change if you wish? Although it fits the style of the rest of the file more like this.

Comment (GovanifY) [reply cmake/SearchForStuff.cmake:206]: No it's fine, I just wanted to know the reasonning, this is why I put it as a comment and not a request for changes

**Review by GovanifY — APPROVED** [review]
LGTM

**Pull request merged**